crawl-ref/source/dat/des/variable/ghost.des

```diff
@@ -0,0 +1,77 @@
+##############################################################################
+# Ghost vaults
+#
+##############################################################################
+
+###############################################################################
+# Two basic ghost vaults
+NAME:   basic_ghost_runed
+CHANCE: 25%
+DEPTH:  D:3-, Lair, Elf, Orc, Snake, Shoals, Swamp, Spider, Depths, Vaults, Crypt, Zot, Dis, Geh, Tar, Coc
+TAGS:   player_ghost luniq_player_ghost chance_player_ghost no_tele_into no_trap_gen allow_dup
+KFEAT:  + = runed_door
+KITEM:  1 = acquire any / any useful good_item
+KMONS:  ! = player ghost
+MAP
+nnnn
++!1n
+nnnn
+ENDMAP
+
+NAME:   basic_ghost_transp
+CHANCE: 25%
+DEPTH:  D:3-, Lair, Elf, Orc, Snake, Shoals, Swamp, Spider, Depths, Vaults, Crypt, Zot, Dis, Geh, Tar, Coc
+TAGS:   player_ghost luniq_player_ghost chance_player_ghost no_tele_into no_trap_gen allow_dup
+KITEM:  1 = acquire any / any useful good_item
+KMONS:  ! = player ghost
+SHUFFLE: 1AQR, PS
+MARKER: P = lua:transp_loc("basic_ghost_transp_entry")
+MARKER: Q = lua:transp_dest_loc("basic_ghost_transp_entry")
+MARKER: R = lua:transp_loc("basic_ghost_transp_exit")
+MARKER: S = lua:transp_dest_loc("basic_ghost_transp_exit")
+SUBST: A = .
+MAP
+ nnnnn
+nn.1.nn
+n.$.$.n
+nA.!.Rn
+n.$.$.n
+nn.Q.nn
+ nnnnn
+ .P@S.
+ENDMAP
+
+NAME:   vaults_ghost_runed
+WEIGHT: 50
+TAGS:   vaults_room player_ghost luniq_player_ghost no_trap_gen no_tele_into allow_dup
+KFEAT:  + = runed_door
+KITEM:  1 = any useful good_item / nothing
+KMONS:  ! = generate_awake player ghost
+SHUFFLE: 123!
```

Comment:
I don't see a 2 or 3 here. Does this mean that this "ghost" vault can generate with or without an item, and with or without a ghost? I'm not sure what the value in that is.